joomla/joomla-cms
# [5.1] Convert mod_feed to service provider

### Summary of Changes

Converts the feed module to service provider.

### Testing Instructions

1. Make sure you have site module mod_feed published and configured
2. You should see module output on frontend

![mod_feed](https://github.com/joomla/joomla-cms/assets/59166858/195bbc20-bd62-4cce-b2c0-0d6d3ca8f1f2)

### Actual result BEFORE applying this Pull Request

Module works as expected

### Expected result AFTER applying this Pull Request

Module works same way as before applying pull request

### Link to documentations
Please select:
